wurst/_handles/DialogBox.wurst

```diff
@@ -0,0 +1,67 @@
+package DialogBox
```

Comment:
Should probably be in ```wurst/util/``` instead of _handles, since it's kinda reserved for primitive, wc3 types.